components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java

```diff
@@ -458,8 +458,9 @@ public OAuth2AccessTokenRespDTO issueAccessToken(OAuth2AccessTokenReqDTO tokenRe
             return tokenRespDTO;
         } catch (Exception e) { // in case of an error, consider it as a system error
             log.error("Error occurred while issuing the access token for Client ID : " +
-                    tokenReqDTO.getClientId() + ", User ID " + tokenReqDTO.getResourceOwnerUsername() +
-                    ", Scope : " + Arrays.toString(tokenReqDTO.getScope()) + " and Grant Type : " +
+                    tokenReqDTO.getClientId() + ", User ID " + (LoggerUtils.isLogMaskingEnable ?
```

Comment:
Do we need to mask client id?

Reply:
No, we are masking the username and password.